region/maridia/inner-pink/Crab Shaft.json

```diff
@@ -403,19 +403,13 @@
     {
       "id": 3,
       "link": [1, 1],
-      "name": "Sciser Farm",
+      "name": "Sciser Farm (1 Sciser)",
       "requires": [
         {"resetRoom": {"nodes": [1]}},
-        {"partialRefill": {"type": "Energy", "limit": 240}},
-        {"partialRefill": {"type": "PowerBomb", "limit": 6}},
-        {"or": [
-          "Gravity",
-          "Spazer",
-          "Wave",
-          "Plasma",
-          "canUseGrapple",
-          "canBePatient"
-        ]}
+        {"cycleFrames": 490}
+      ],
+      "farmCycleDrops": [
+        {"enemy": "Sciser", "count": 1}
```

Comment:
This is a no item 2 farm.
No items + no tech isn't trivial to 1 farm either but maybe leniency makes that problem go away.